packages/x-date-pickers-pro/src/themeAugmentation/components.d.ts

```diff
@@ -13,29 +13,31 @@ export interface PickersProComponents<Theme = unknown> {
     defaultProps?: ComponentsProps['MuiDateRangePickerToolbar'];
     styleOverrides?: ComponentsOverrides<Theme>['MuiDateRangePickerToolbar'];
   };
-  MuiDesktopNextDateRangePicker?: {
-    defaultProps?: ComponentsProps['MuiDesktopNextDateRangePicker'];
-    styleOverrides?: ComponentsOverrides<Theme>['MuiDesktopNextDateRangePicker'];
-  };
-  MuiMobileNextDateRangePicker?: {
-    defaultProps?: ComponentsProps['MuiMobileNextDateRangePicker'];
-    styleOverrides?: ComponentsOverrides<Theme>['MuiMobileNextDateRangePicker'];
-  };
   MuiMultiInputDateRangeField?: {
     defaultProps?: ComponentsProps['MuiMultiInputDateRangeField'];
     styleOverrides?: ComponentsOverrides['MuiMultiInputDateRangeField'];
   };
-  MuiNextDateRangePicker?: {
-    defaultProps?: ComponentsProps['MuiNextDateRangePicker'];
-    styleOverrides?: ComponentsOverrides<Theme>['MuiNextDateRangePicker'];
-  };
   MuiSingleInputDateRangeField?: {
     defaultProps?: ComponentsProps['MuiSingleInputDateRangeField'];
     styleOverrides?: ComponentsOverrides['MuiSingleInputDateRangeField'];
   };
-  MuiStaticNextDateRangePicker?: {
-    defaultProps?: ComponentsProps['MuiStaticNextDateRangePicker'];
-    styleOverrides?: ComponentsOverrides<Theme>['MuiStaticNextDateRangePicker'];
+
+  // Date Range Pickers
```

Comment:
Do you think using sections makes more sense than alphabetical order? 😃

Reply:
In the community package, I think so.
We had legacy pickers missing from this page because it's super long and it's hard to find mistake on it.
That's why I grouped it, if we have a picker with less than 4 elements, something is wrong.

For the pro package, I don't think it brings much value indeed.
I went for consistency, because I don't think it makes the file less readable.

Reply:
> We had legacy pickers missing from this page because it's super long and it's hard to find mistake on it.
> That's why I grouped it, if we have a picker with less than 4 elements, something is wrong.

Mhh, indeed. This file is not trivial to manage. 🙈 
Grouping by component types probably makes more sense. 👍